ophyd/positioner.py

```diff
@@ -152,7 +147,8 @@ def low_limit(self):
     def high_limit(self):
         return self.limits[1]
 
-    def move(self, position, moved_cb=None, timeout=None):
+    @abstractmethod
+    def move(self, position, wait=False, moved_cb=None, timeout=None):
```

Comment:
I'm not sure what to do about the argument order here.

Adding `wait` as the second positional arguement matches the order of `EpicmMotor.move` and `PseudoPositioner.move` but doing so is a major API breakage on this method (as if anyone was passing `moved_cb` positionally then they would both not get there callback run but wait for the motion to finish!

Reply:
I had the same dilemma. I thought already implemented `move` show how everybody is supposed to have it.

What about adding `*` after `position` to force use of keyword args ? 

I agree it is an API change. On the other hand current situation is not ideal neither. 

Another possibility is to move `wait` argument at the end...